pkg/extract/util/test/test.go

```diff
@@ -142,3 +142,45 @@ func Diff(t *testing.T, expectedAbsPath, gotAbsPath string) {
 		}
 	}
 }
+
+// QueryUnescapeFileTree copies a file tree at "from" to <temp-dir>/"to" by query-unscapeing file names.
+// It returns <temp-dir>/"to".
+func QueryUnescapeFileTree(t *testing.T, from, to string) string {
+	toPath := filepath.Join(t.TempDir(), to)
```

Comment:
It feels strange that "from" indicates a path, but "to" is a directory name.

Is it not possible to receive the argument `to` as a path like this?
```
QueryUnescapeFileTree(t, from, filepath.Join(t.TempDir(), to))
```

Reply:
I had a little bad feeling at the point, too.
This suggested change will eliminate the necessity of return value.
I will take it!

Reply:
Thinking...

This change leaves redundant code... ugly...

```
			inputDir := filepath.Join(t.TempDir(), "vuls-data-raw-oracle")
			utiltest.QueryUnescapeFileTree(t, tt.fixturePath, inputDir)
			outputDir := t.TempDir()
			err := oracle.Extract(inputDir, oracle.WithDir(outputDir))
```

It's possible that let QueryUnescapeFileTree return inputDir, but it seems unnatural to return one of arguments as it is. Hmm...

Reply:
Changed the argument names instead. How's that?

```
// QueryUnescapeFileTree copies a file tree at fixturePath to <tmp-dir>/rawGroup by query-unescaping file names.
// It returns <tmp-dir>/rawGroup.
func QueryUnescapeFileTree(t *testing.T, fixturePath, rawGroup string) string {
	rawPath := filepath.Join(t.TempDir(), rawGroup)
```